docs/package.json

```diff
@@ -1,33 +1,37 @@
 {
   "name": "remeda-docs",
   "version": "1.0.0",
-  "main": "index.js",
   "repository": "git@github.com:remeda/docs.git",
   "author": "Łukasz Sentkiewicz",
   "license": "MIT",
   "scripts": {
-    "start": "node index",
-    "watch": "parcel serve index.html",
-    "build": "cross-env NODE_ENV=production parcel build index.html --public-url ./  --out-file index.html && cp _redirects dist/_redirects && cp dist/index.html dist/404.html"
+    "watch": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "start": "vite preview"
   },
   "dependencies": {
-    "@types/react-router-dom": "^4.3.0",
-    "compression": "^1.7.2",
-    "connect-history-api-fallback": "^1.5.0",
-    "express": "^4.17.3",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2",
-    "react-router": "^4.3.1",
-    "react-router-dom": "^4.3.1",
-    "react-syntax-highlighter": "^7.0.4"
+    "compression": "^1.7.4",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router": "^6.15.0",
+    "react-router-dom": "^6.15.0",
+    "react-syntax-highlighter": "^15.5.0"
   },
   "devDependencies": {
-    "@types/react": "^16.3.14",
-    "@types/react-dom": "^16.0.5",
-    "@types/react-syntax-highlighter": "^0.0.5",
-    "cross-env": "^5.1.5",
-    "node-sass": "^8.0.0",
-    "parcel": "^1.9.4",
-    "typescript": "^3.8.3"
+    "@types/react": "^18.2.20",
+    "@types/react-dom": "^18.2.7",
+    "@types/react-router-dom": "^5.3.3",
+    "@types/react-syntax-highlighter": "^15.5.7",
+    "@typescript-eslint/eslint-plugin": "^6.0.0",
+    "@typescript-eslint/parser": "^6.0.0",
+    "@vitejs/plugin-react": "^4.0.3",
+    "cross-env": "^7.0.3",
+    "eslint": "^8.45.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
you install this plugin but don't use it

Reply:
cross-env, @typescript-eslint/* , eslint, these modules can be removed. Change eslint config from root to extend